tests/pytest/ambit/test_ambit.py

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-import forte
-
-def test_ambit():
-    """Test the conversion of ambit::Tensor to numpy"""
-    t = forte.test_ambit_3d()
```

Comment:
There must be a function called `test_ambit_3d()`. It looks like it will be no longer needed. Can you please remove it?

Reply:
[Already done.](https://github.com/evangelistalab/forte/pull/290/files#diff-8a41d74eddc62044d7b9cd2b0f5817cb68719d19f9acdc4ae5b7b3ed0e6ed90eL68-L82)